ml-agents/mlagents/trainers/learn.py

```diff
@@ -320,26 +321,29 @@ def run_training(run_seed: int, options: RunOptions) -> None:
     :param run_options: Command line arguments for training.
     """
     with hierarchical_timer("run_training.setup"):
-        model_path = f"./models/{options.run_id}"
+        base_path = "results"
+        write_path = os.path.join(base_path, options.run_id)
         maybe_init_path = (
-            f"./models/{options.initialize_from}" if options.initialize_from else None
+            os.path.join(base_path, options.run_id) if options.initialize_from else None
         )
-        summaries_dir = "./summaries"
+        run_logs_dir = os.path.join(write_path, "run_logs")
```

Comment:
You could use `pathlib` for some of these operations.  For example..

```python
base_path = Path('./results')
write_path = base_path / options.run_id
maybe_init_path = base_path / options.run_id if options.initialize_from else None
run_logs_dir = base_path / run_logs
run_logs_dir.mkdir(exist_ok=True)
```

Not a big deal, but I do think it is a little nicer than the `os` and `glob` equivalents.

Reply:
Tried this, liked it, then realized Tensorflow's utilities don't support Path objects, and backed out as we'd have a bunch of Path to `str` conversions that are easy to mess up. Another thing to do once we convert to PyTorch...

Reply:
OK -- no worries.  Let's just refactor to use it when it makes sense